build.gradle.kts

```diff
@@ -1,11 +1,9 @@
 plugins {
-    kotlin("multiplatform") version "1.7.20" apply false
-    id("org.jetbrains.kotlinx.kover") version "0.6.1" apply false
+    kotlin("multiplatform") version "1.8.10" apply false
+    id("org.jetbrains.kotlinx.kover") version "0.7.0-Alpha" apply false
```

Comment:
Shall we use kover alpha? or it is not compatible with new Kotlin version?

Reply:
well, it is the latest version for Kover.

Reply:
yeah, what I meant was that maybe we should stop using them until they get a bit more mature.

Reply:
Oh Okay, I see what you mean. Jacoco is not that multiplatform friendly either.